packages/button/src/LionButton.js

```diff
@@ -67,24 +52,35 @@ export class LionButton extends DisabledWithTabIndexMixin(SlotMixin(LitElement))
     return [
       css`
         :host {
+          position: relative;
           display: inline-block;
           min-height: 40px; /* src = https://www.smashingmagazine.com/2012/02/finger-friendly-design-ideal-mobile-touchscreen-target-sizes/ */
-          outline: 0;
-          background-color: transparent;
           box-sizing: border-box;
+          vertical-align: middle;
+          line-height: 24px;
+          background: #eee; /* minimal styling to make it recognizable as btn */
+          padding: 8px; /* padding to fix with min-height */
+          outline: none; /* focus style handled below */
         }
 
-        .btn {
-          min-height: 24px;
+        :host::after {
+          position: absolute;
+          content: '';
+          top: 0;
+          right: 0;
+          bottom: 0;
+          left: 0;
+          margin: 0;
+          padding: 0;
+        }
+
+        .button-content {
```

Comment:
Not sure if it would be possible to omit this layer and apply flex-inline on the parent?

Reply:
The slotted svg cause some problems with vertical align. I couldn't get a better approach. I'm open to suggestions.

Currently if you put a button inline with other with icon you can check the misalignment

Reply:
Yep, we cannot omit this layer because of IE and this needs to be the flex-parent for the content